runtime/vm/LayoutFFITypeHelpers.cpp

```diff
@@ -203,7 +203,13 @@ LayoutFFITypeHelpers::getStructFFITypeElements(char **layout)
 void
 LayoutFFITypeHelpers::freeStructFFIType(ffi_type *ffiType)
 {
-	if ((NULL != ffiType) && (FFI_TYPE_STRUCT == ffiType->type)) {
+	if ((NULL != ffiType)
+			&& ((FFI_TYPE_STRUCT == ffiType->type)
+#if defined(J9ZOS390)
+				|| (FFI_TYPE_STRUCT_FF == ffiType->type)
+				|| (FFI_TYPE_STRUCT_DD == ffiType->type)
+#endif /* defined(J9ZOS390) */
+	 )) {
```

Comment:
Please remove the space before `))` (there should be just one tab).

Reply:
Fixed in https://github.com/eclipse-openj9/openj9/pull/19645/commits/38245ec2872c2b2664f6fdd3947f80e71e63c57d#diff-7e388e6ac3235f9470ab3d7607ffbe1c8fceacddd69ac992fdae6cda3e99257d